profiles/lacuna_stories/lacuna_stories.info

```diff
@@ -3,7 +3,7 @@ name = "Lacuna Stories"
 description = "Installation profile for Lacuna Stories."
```

Comment:
Well, we can't change feature and module names and such without a lot of work, you're right. But the description does get shown to users, I think. Either way, it's not machine name stuff, so it's good.